src/hb-ot-layout-common.hh

```diff
@@ -646,8 +646,7 @@ struct FeatureParamsCharacterVariants
       return;
```

Comment:
Do we want to also drop the check for `numNamedParameters >= 0x7FFF` above?

Reply:
Looks suspicious indeed.

Reply:
oh yeah, I think I just ported it from old fonttools' code

Reply:
I'll remove this check in another PR, I'm going to add a testcase from Cosimo

Reply:
I found another issue in fonttools related with FeatureParams, so I'll add a testcase later after that got fixed